tests/e2e/raw_block_volume.go

```diff
@@ -118,13 +128,6 @@ var _ = ginkgo.Describe("raw block volume support", func() {
 		ctx, cancel := context.WithCancel(context.Background())
 		defer cancel()
 
-		ginkgo.By(fmt.Sprintf("Deleting all statefulsets in namespace: %v", namespace))
```

Comment:
if the test case crashes in the middle it will not get to run its defer calls, that is why we put all the cleanups in `Aftereach()` as well

Reply:
reverted to original change